Source/Scene/Cesium3DTile.js

```diff
@@ -61,30 +67,69 @@ define([
         var contentHeader = header.content;
 
 // TODO: For the 3D Tiles spec, we want to change this to basically (x, y, z) to (x, y, z)
-        var b = header.box;
-        var rectangle = new Rectangle(b[0], b[1], b[2], b[3]);
-
-        this._tileBoundingBox = new TileBoundingBox({
-            rectangle : rectangle,
-            minimumHeight : b[4],
-            maximumHeight : b[5]
-        });
-        this._orientedBoundingBox = OrientedBoundingBox.fromRectangle(rectangle, b[4], b[5]);
+        var tileBoundingVolume = header.boundingVolume;
+        if (defined(tileBoundingVolume.box)) {
```

Comment:
@pjcozzi I went over that a bit too fast. I agree about `box` and have made some changes in https://github.com/AnalyticalGraphicsInc/cesium/commit/c7e9c9b23e99ba039e33efd274e90b90c983fbac, but I'm not confident that my usage of `halfAxes` in `OrientedBoundingBox` is right (see [Cesium3DTile](https://github.com/AnalyticalGraphicsInc/cesium/blob/c7e9c9b23e99ba039e33efd274e90b90c983fbac/Source/Scene/Cesium3DTile.js#L277-L285) and [TileOrientedBoundingBox](https://github.com/AnalyticalGraphicsInc/cesium/commit/c7e9c9b23e99ba039e33efd274e90b90c983fbac#diff-e9edfe6c30a08cde0c705cc817c945c5R18)). Does this look reasonable?